spec-tests/src/harness/config/index.ts

```diff
@@ -0,0 +1,46 @@
+import {getEnv} from "./getEnv";
+import assert from "node:assert";
+import {privateKeyToAccount} from "viem/accounts";
+import {Account} from "viem";
+
+export function loadHarnessConfig():HarnessConfig {
+  const env = getEnv()
+  const nodeType = env.string('HARNESS_NODE')
+  assert(nodeType === 'anvil' || nodeType === 'tenderly', 'HARNESS_NODE must be either anvil or tenderly')
+
+  const node: HarnessNode =
+    nodeType === 'anvil'
+      ? {
+        mode: 'anvil',
+        alchemyApiKey: env.string('ALCHEMY_API_KEY'),
+      }
+      : {
+        mode: 'tenderly',
+        tenderlyApiKey: env.string('TENDERLY_API_KEY'),
+        tenderlyAccount: env.string('TENDERLY_ACCOUNT'),
+        tenderlyProject: env.string('TENDERLY_PROJECT'),
+      }
+
+  return {
+    node,
+    sender:  privateKeyToAccount('0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80') // anvil default #1
```

Comment:
that's a good point with predictable sender 👍 
I think that anvil and hardhat node create test accounts with some predefined seed phrase. I could check with the team is it possible that tenderly also does "predictable private keys creation"

Reply:
I woudn't worry about this too much. This was only refactoring and it works fine in tendely -- we can always mint some ETH if we need it.